tempest/api/workloadmgr/base.py

```diff
@@ -3189,14 +3189,16 @@ def create_restore_json(self, rest_details):
             LOG.debug("Network details for restore: " + str(network_details))
             instances = rest_details['instances']
             instance_details = []
+            if rest_details['volume_type'] == None:
```

Comment:
Existing test scripts do not pass volume_type details in rest_details dict. This will throw KeyError. Please fix it.

Reply:
I have updated the create_restore_json function to accept volume_type parameter